__tests__/index.test.js

```diff
@@ -139,4 +139,52 @@ describe("/api/articles/:articles_id", () => {
         expect(response.body.message).toEqual("invalid id type");
       });
   });
+
+  test("responds with an array of comments for the given article_id ", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBeGreaterThan(0);
+        response.body.comments.forEach((comment) => {
```

Comment:
If in the future you were to change your test data this test might fail, try to write more robust tests. You can check the type of the value as that will be consistent :)